src/App/App.js

```diff
@@ -26,22 +32,33 @@ class App extends Component {
 
   render() {
     return (
-      <div>
-        {this.props.isLoggedIn ? (
-          <button onClick={this.handleLogout}>Log out</button>
-        ) : null}
-        <AuthorizationPage></AuthorizationPage>
+      <Router>
+        <Switch>
+          <Route exact path="/home" component={HomePage} />
+          <Route path="/exercise" component={ExercisePage} />
+          <Route path="/nutrition" component={NutritionPage} />
+          <Route path="/community" component={CommunityPage} />
+          <Route path="/user" component={UserPage} />
+          <Route path="/authorization" component={AuthorizationPage} />
+        </Switch>
+        <div>
+          {this.props.isLoggedIn ? (
+            <button onClick={this.handleLogout}>Log out</button>
+          ) : null}
+          <AuthorizationPage></AuthorizationPage>
 
-        {/* Below is a section for playing with new components, that can be hidden or shown by clicking the button below in the App.  I thought it might help to have it separate until new components are ready to be added to their appropriate place? idk... Daniel */}
-        <button onClick={this.handleHidePlayground}>
-          {this.state.playground ? "Hide Playground" : "Show Playground"}
-        </button>
-        {this.state.playground ? (
-          <div className="component_playground">
-            <SmokeButton></SmokeButton>
-          </div>
-        ) : null}
-      </div>
+          {/* Below is a section for playing with new components, that can be hidden or shown by clicking the button below in the App.  I thought it might help to have it separate until new components are ready to be added to their appropriate place? idk... Daniel */}
+          <button onClick={this.handleHidePlayground}>
+            {this.state.playground ? "Hide Playground" : "Show Playground"}
+          </button>
+          {this.state.playground ? (
+            <div className="component_playground">
+              <SmokeButton></SmokeButton>
+              <NavigationComponent></NavigationComponent>
```

Comment:
Do you feel like this component is ready to be put into the main app? I feel like it's ready.  If you agree, please replace line 48, ```<AuthorizationPage></AuthorizationPage>``` with your component.

Reply:
Also, sorry, I just noticed there are merge conflicts with the most recent version of dev, caused mainly by the most recent merge into dev from the news feed component.

Reply:
K I replaced it with the nav component. I can handle the merge conflicts while I'm merging right?

Reply:
yeah, you're right.